relay/nakama/auth/siwe.go

```diff
@@ -34,48 +34,87 @@ func validateSIWE(signer, message, signature string) (isValidationRequest bool,
 	return isValidationRequest, nil
 }
 
-func handleSIWE(
+func processSIWE(
 	ctx context.Context,
 	logger runtime.Logger,
 	nk runtime.NakamaModule,
-	in *api.AuthenticateCustomRequest,
-) (
-	*api.AuthenticateCustomRequest, error,
-) {
-	signerAddress := in.GetAccount().GetId()
-	signature := in.GetAccount().GetVars()["signature"]
-	message := in.GetAccount().GetVars()["message"]
-
+	signerAddress, signature, message string,
+) (bool, error) {
```

Comment:
You don't need to return a boolean, you can see whether or not the processing is successful by just checking whether or not err is nil.

Reply:
updated